.ci/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-version=$(awk '/^ENV RAPIDPRO_VERSION/ { print $3 }' rapidpro-docker/Dockerfile)
+version=$(curl -s https://api.github.com/repos/praekeltfoundation/rapidpro/tags  | jq -r '.[0].name')
```

Comment:
I think it is fine, the DockerFile uses that to download from github. `RAPIDPRO_VERSION` comes from the curl command above.
```RUN echo "Downloading RapidPro ${RAPIDPRO_VERSION} from https://github.com/$RAPIDPRO_REPO/archive/${RAPIDPRO_VERSION}.tar.gz" && \
    wget -O rapidpro.tar.gz "https://github.com/$RAPIDPRO_REPO/archive/${RAPIDPRO_VERSION}.tar.gz" && \
    tar -xf rapidpro.tar.gz --strip-components=1 && \
    rm rapidpro.tar.gz